### ISSUE TITLE
Requirements for Running Symfony

### ISSUE THREAD

**Issue by Shoplifter** [symfony/symfony-docs#10729]
The [Requirements for Running Symfony](https://symfony.com/doc/current/reference/requirements.html) always says "Symfony **4.0** requires PHP 7.1.3 or higher to run" - no matter which 4.* version you have selected in the dropdown menu. So it's not shown which PHP version actually is required for Symfony 4.1+

**Comment by xabbuh** [comment]
I think we should reword this to read "Symfony 4 requires [...]". Would you like to send a pull request targetting the `4.1` branch?
